pdp/dao.py

```diff
@@ -30,28 +30,56 @@ def get_profile(netid):
     )
     # TODO: We eventually won't pass in fake_profile
     profile = Profile(dct=fake_profile)
+
+    person = get_person(netid)
+    if 'hepps' or 'uwhr' in person.identifiers:
```

Comment:
Yes...just spent some time playing with the interpreter and have my head wrapped around it now.  I fix.